assets/src/edit-story/app/story/useStoryReducer/reducers/addPage.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Insert page as the new last page.
+ *
+ * Current page will be updated to point to the newly inserted page.
+ *
+ * Selection is cleared.
+ *
+ * @param {Object} state Current state
+ * @param {Object} payload Action payload
+ * @param {Object} payload.page Object with properties of new page
+ * @return {Object} New state
+ */
+function addPage( state, { page } ) {
```

Comment:
Nit, but we will likely have `addPage` at `position` (or after `position`/`page`) in short order. Ok to leave until later.

Reply:
Let's push back to @samitron7. I'll ask in the Figma.

"Does adding a page add the new page after the current page or only to the end of the list of pages?"

Reply:
@samitron7 confirms, that pressing "add page" adds a blank page *directly after the current page*. So maybe this underlying function should do exactly that? Or should it be `addPageAtPosition`?

Reply:
imho, just having `{page, position}` as payload would be good enough?

Reply:
Yes, there's now [`addPage`](https://github.com/ampproject/amp-wp/pull/3910/files#diff-30af242e1170b1c57b72a85d7d353543R6-R7) assuming _"add right after current page"_ and [`addPageAt`](https://github.com/ampproject/amp-wp/pull/3910/files#diff-30af242e1170b1c57b72a85d7d353543R9-R10) taking a position.

Also see [the underlying implementation](https://github.com/ampproject/amp-wp/pull/3910/files#diff-2140a3febf9327cf423699f615121608R21-R37).